README.md

```diff
@@ -163,8 +163,12 @@ After about 30-60 seconds the crons should have completed and you should have da
 
 ## Testing
 
+The application currently uses two frameworks for front-end testing: [Cypress](https://www.cypress.io/) and [Jest](https://jestjs.io/). There are plans to implement back-end tests in the future.
```

Comment:
I think you need to sync you remote master with Upstream master as reflected from here